src/Umbraco.Infrastructure/BackgroundJobs/RecurringBackgroundJobHostedServiceRunner.cs

```diff
@@ -33,45 +26,45 @@ public class RecurringBackgroundJobHostedServiceRunner : IHostedService
 
     public async Task StartAsync(CancellationToken cancellationToken)
     {
-        _logger.LogInformation("Creating recurring background jobs hosted services");
-
-        // create hosted services for each background job
-        _hostedServices = _jobs.Select(_jobFactory).ToList();
-
         _logger.LogInformation("Starting recurring background jobs hosted services");
 
-        foreach (IHostedService hostedService in _hostedServices)
+        foreach (IRecurringBackgroundJob job in _jobs)
         {
+            var jobName = job.GetType().Name;
             try
             {
-                _logger.LogInformation($"Starting background hosted service for {hostedService.GetType().Name}");
+
+                _logger.LogInformation("Creating background hosted service for {job}", jobName);
```

Comment:
I think this is a bit too much spam, there's already a "Starting ...", and if it fails a "Failed to start ..." message, there are no other possibilities.

Reply:
Creating a job and starting it can have 2 different failure reasons.
With the current code flow, if you see a `failed to start` message it might be because of the factory failing or the created service not actually starting.

A way to resolve this and reduce messages being logged is to Log a message right after getting the jobName `Booting/Initializing/Hosting background job {Jobname}`
And wrapping the factory and the start in separate try catches with each having a distinct message template
For Completnes, you could leave the Creating/Starting messages but change their level to debug.